tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -959,6 +1028,12 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Variables", "[DataStructures][Unit]") {
     test_variables_assign_subset<DataVector>();
     test_variables_assign_subset<ModalVector>();
   }
+  SECTION("Test Variables extract subset") {
```

Comment:
All of the `SECTION`s here should probably be `INFO`s. I don't expect you to change it in this PR though.

Reply:
Did this result in an issue? If not, could one please be created?

Reply:
It's not how I would have written it, but I don't think there is anything wrong with the way the code is currently because there is nothing outside the SECTIONs.

Reply:
I'm fine with that. @nilsleiffischer if it really matters to you could you put in a PR please?

Reply:
Depends - is there any drawback in keeping the `SECTION`s as-is, e.g. in test runtime?

Reply:
Not that I have ever tried to measure or noticed. It doesn't restart the executable, just does a new function call, so it should be negligible.